pkg/proxy/proxy.go

```diff
@@ -96,15 +97,20 @@ type Proxy struct {
 	// Datapath updater for installing and removing proxy rules for a single
 	// proxy port
 	datapathUpdater DatapathUpdater
+
+	// IPCache is used for tracking IP->Identity mappings and propagating
+	// them to the proxy via NPHDS in the cases described
+	ipcache *ipcache.IPCache
```

Comment:
nit: this var is shadowing the package name. Maybe a quick rename?

Reply:
This is embedded into the structure, so no shadowing here. I couldn't think of any reasonable renames either, unless we want to get into more obtuse naming like `ipc` or something.